src/plugins/intel_gpu/tests/unit/transformations/swiglu_decomposition_test.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2024 Intel Corporation
```

Comment:
nit: the file should probably be named `swiglu_fusion_test`

Reply:
renamed